main.py

```diff
@@ -3147,21 +3147,26 @@ async def fake(message: discord.Interaction):
 @bot.tree.command(description="(ADMIN) Force cats to appear")
 @discord.app_commands.default_permissions(manage_guild=True)
 @discord.app_commands.rename(cat_type="type")
-@discord.app_commands.describe(cat_type="select a cat type ok")
+@discord.app_commands.describe(cat_type="select a cat type ok", channel="another channel then")
 @discord.app_commands.autocomplete(cat_type=cat_type_autocomplete)
-async def forcespawn(message: discord.Interaction, cat_type: Optional[str]):
+async def forcespawn(message: discord.Interaction, cat_type: Optional[str], channel: Optional[discord.TextChannel]):
     if cat_type and cat_type not in cattypes:
         await message.response.send_message("bro what", ephemeral=True)
         return
-
+    channel_id = message.channel.id
```

Comment:
```suggestion
    if not channel:
        channel_id = message.channel.id
```

Reply:
I don't know how to handle optional arguments, I was afraid of an error happening